components/create-post-page/TextbookUploaded.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  Box,
+  Button,
+  Container,
+  HStack,
+  Icon,
+  Link,
+  Spacer,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import { UserIcon } from "@heroicons/react/solid";
+import { PostWithBookWithUser } from "@lib/services/post";
+import PostCard from "../PostCard";
+
+interface Props {
+  post: PostWithBookWithUser;
+}
+
+export default function TextbookUploaded({ post }: Props) {
+  return (
+    <Container mw="container.md">
```

Comment:
Ambiguous reading this on whether this is min or max width, consider `maxWidth` instead.